### PR TITLE
Link the actual #ricing-theming channel in a clickable way

### PR DESCRIPTION
**Pull request by eepykate**
This is hard-coded and doesn't use an environment variable,
but so was the channel name, so it doesn't really change
much in that area.

Update: doesn't hard-code anymore... after much pain and suffering.

### CODE REVIEW

**Comment by eepykate** [comment]
well i fucked up

how the fuck do I use variables??

**Comment by eepykate** [comment]
if someone wants to non-hardcode it, be my guest. I am not fucking with variables in this thing. @elkowar :angry: why did you make me try that

**Comment by eepykate** [comment]
4 force pushes later... Jesus... This was unnecessarily hard.

File: main.go [hunk 1]
```diff
@@ -153,7 +154,7 @@ func messageCreate(s *discordgo.Session, m *discordgo.MessageCreate) {
 				return
 			}
 
-			s.ChannelMessageSend(ch.ID, "Your showcase submission was detected to be invalid. If you wanna comment on a rice, use the #ricing-theming channel.\nIf this is a mistake, contact the moderators or open an issue on https://github.com/unixporn/trup")
+			s.ChannelMessageSend(ch.ID, fmt.Sprintf("Your showcase submission was detected to be invalid. If you wanna comment on a rice, use the <#%s> channel.\nIf this is a mistake, contact the moderators or open an issue on https://github.com/unixporn/trup", env.ChannelModlog))
```
Comment (mlvzk) [inline main.go]:
```suggestion
			s.ChannelMessageSend(ch.ID, fmt.Sprintf("Your showcase submission was detected to be invalid. If you wanna comment on a rice, use the <#%s> channel.\nIf this is a mistake, contact the moderators or open an issue on https://github.com/unixporn/trup", env.ChannelRicing))
```

Comment (eepykate) [reply main.go]: oof, whoops.

**Pull request closed without merging**